packages/polyfill-library/polyfills/Symbol/replace/config.json

```diff
@@ -4,7 +4,9 @@
 		"es2015"
 	],
 	"browsers": {
-		"ie": "< 13",
+		"edge": "*",
```

Comment:
This is likely incorrect, but I would rather serve a polyfill to a browser that doesn't need it than to not serve a polyfill to a browser that does. In a subsequent pull-request I will add a script that checks if polyfills are being served to browsers which do not require them, and I will have the script run in CircleCI, causing builds to break if it finds such a situation.